android/lint.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lint>
+    <!-- list of issues to configure -->
+
+    <!-- Downgrade severity of not explicitly including all options in switch statements.  (using default does not eliminate this warning) -->
+    <issue id="SwitchIntDef" severity="weak warning"/>
+
+    <!-- Upgrade severity of using wrong app compat method - want to highlight cases where this may be occurring. -->
+    <issue id="AppCompatMethod" severity="error"/>
+
+    <!-- Upgrade severity of not using the Firebase App Indexing API - we may not need this one, as we already have “Missing support for Firebase App Indexing” enabled -->
+    <issue id="GoogleAppIndexingApiWarning" severity="warning"/>
+
+    <!-- Enable checking for app links -->
+    <issue id="AppLinksAutoVerifyError" severity="warning"/>
+
+    <!-- Enable checking for potential app links -->
+    <issue id="AppLinksAutoVerifyWarning" severity="warning"/>
+
+    <!-- Enable weak checking for UI design back button issues -->
+    <issue id="BackButton" severity="weak warning"/>
+
+    <!-- Upgrade severity of trying to specify AdapterView children in XML -->
+    <issue id="AdapterViewChildren" severity="error"/>
+
+    <!-- Upgrade severity of including more than one layout within a ScrollView -->
+    <issue id="ScrollViewCount" severity="error"/>
+
+    <!-- Enable checking for implied unsupported Chrome OS hardware -->
+    <issue id="PermissionImpliesUnsupportedChromeOsHardware" severity="warning"/>
+
+    <!-- Enable checking for unsupported Chrome OS hardware -->
+    <issue id="UnsupportedChromeOsHardware" severity="error"/>
+
+    <!-- Enable checking for insecure cipher mode -->
+    <issue id="GetInstance" severity="error"/>
+
+    <!-- Check for easter eggs -->
+    <issue id="EasterEgg" severity="weak warning"/>
+
+    <!-- Look for STOPSHIP comments -->
+    <issue id="StopShip" severity="error"/>
+
+    <!-- Look for mangled file line endings -->
+    <issue id="MangledCRLF" severity="weak warning"/>
+
+    <!-- Increase severity of incorrect Gradle paths -->
+    <issue id="GradlePath" severity="error"/>
+
+    <!-- Notify of newer library versions -->
+    <issue id="NewerVersionAvailable" severity="info"/>
+
+    <!-- Enable checking for incorrect icon sizes -->
+    <issue id="IconExpectedSize" severity="warning"/>
+
+    <!-- Enable WebP icon conversion suggestion -->
+    <issue id="ConvertToWebp" severity="info"/>
+
+    <!-- Upgrade severity of this check to improve layout computation speed -->
+    <issue id="DisableBaselineAlignment" severity="error"/>
+
+    <!-- Add support for checking for logging calls in production code -->
+    <issue id="LogConditional" severity="weak warning"/>
```

Comment:
What do you think we should do about this one?  Leave it as a weak warning, or eliminate the check for now?